travis_setup.php

```diff
@@ -128,7 +140,20 @@
 
 run("tar -cf $moduleArchivePath * .??*");
 
-run("git clone --depth=100 --quiet -b $coreInstallerBranch git://github.com/silverstripe/silverstripe-installer.git $targetPath");
+run("wget $installerDist -O /tmp/installer.zip");
+
+$zip = new ZipArchive();
+$zip->open('/tmp/installer.zip');
+$folderName = $zip->getNameIndex(0);
+
+run("unzip /tmp/installer.zip -d /tmp/");
+
+run("mkdir -p $targetPath");
+
+run("mv /tmp/{$folderName}/* $targetPath");
+run("mv /tmp/{$folderName}/.* $targetPath");
+
+die;
```

Comment:
`git clone --branch` can take a tag. It would be a lot less code than all of this.